Drop-In/src/main/java/com/braintreepayments/api/DropInUIEventType.java

```diff
@@ -0,0 +1,16 @@
+package com.braintreepayments.api;
+
+import androidx.annotation.IntDef;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.SOURCE)
+@IntDef({
+        DropInUIEventType.SHOW_VAULT_MANAGER,
+        DropInUIEventType.DID_DISPLAY_SUPPORTED_PAYMENT_METHODS,
+})
+@interface DropInUIEventType {
+    int SHOW_VAULT_MANAGER = 0;
+    int DID_DISPLAY_SUPPORTED_PAYMENT_METHODS = 2;
```

Comment:
Is there another event that is going to be added as 1 or should these be 0 and 1?

Reply:
Good call! I'll update the ordering to make it make more sense.